todoist_export.py

```diff
@@ -16,25 +16,54 @@ def __init__(self, token: str):
         self.cache = {}
 
     def get_completed_activities(self, from_dt: datetime, until_dt: datetime) -> List:
+        """get completed activities from todoist api
+
+        Args:
+            from_dt (datetime): from datetime (timezone aware)
+            until_dt (datetime): until datetime (timezone aware)
+
+        Returns:
+            List: list of completed activities
+        """
         activities = []
         # params: https://developer.todoist.com/sync/v8/?shell#get-activity-logs
-        # TODO: support more than 100 events
-        # activities are ordered from latest
-        data = self.api.activity.get(
-            object_type="item", event_type="completed", limit=100
+        # calculate diff according 'page' param of api
+        tz = from_dt.tzinfo
+        now: datetime = datetime.now(tz=tz)
+        this_monday = now.date() - timedelta(days=now.weekday())
+        from_dt_monday = from_dt.date() - timedelta(days=from_dt.weekday())
+        from_dt_delta_week = int(
+            (this_monday - from_dt_monday).total_seconds() / (7 * 24 * 3600)
         )
-        if "error" in data:
-            self.logger.error(
-                "todoist api returned error: {}".format(data["error_tag"])
+        until_dt_monday = until_dt.date() - timedelta(days=until_dt.weekday())
+        until_dt_delta_week = int(
+            (this_monday - until_dt_monday).total_seconds() / (7 * 24 * 3600)
+        )
+        # if future date specified, set page as 0 (curent week)
+        until_dt_delta_week = 0 if until_dt_delta_week < 0 else until_dt_delta_week
```

Comment:
calculating week difference

X: |--(from)------------|
2 : |------(until)--------|
1 : |--------------|
0 : |-----| (now)

args could be like upper